dashboard/app/controllers/aichat_controller.rb

```diff
@@ -47,6 +47,27 @@ def chat_completion
     sagemaker_response = AichatSagemakerHelper.request_sagemaker_chat_completion(input, params[:aichatModelCustomizations][:selectedModelId])
     latest_assistant_response = AichatSagemakerHelper.get_sagemaker_assistant_response(sagemaker_response)
 
+    filter_result = ShareFiltering.find_failure(latest_assistant_response, locale)
+    if filter_result&.type == ShareFiltering::FailureType::PROFANITY
+      # log here that profanity happened
+      new_messages = [
+        {
+          role: 'user',
+          content: params[:newMessage],
+          status: SharedConstants::AI_INTERACTION_STATUS[:ERROR]
+        },
+      ]
+
+      return render(
+        status: :ok,
+        json: {
+          status: SharedConstants::AICHAT_ERROR_TYPE[:PROFANITY_MODEL],
+          error: 'Profane content generated by model',
```

Comment:
Are we using this `error` value on the front end?

Reply:
Nope! Removed.